### PR TITLE
Added HW20

### PR DESCRIPTION
**Pull request by Pavel-Gulida**

### CODE REVIEW

File: HW20/main.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   if len(middle_name) > 0:
+       full_name = first_name + ' ' + middle_name + ' ' + last_name
+   else:
+       full_name = first_name + ' ' + last_name
+   return full_name.title()
+
+
+class TestFibonacci(unittest.TestCase):
+    def test_fibonacci_equal(self):
+        f = Fibonacci()
```
Comment (mikolaz27) [inline HW20/main.py:33]: split by different tests, add corner cases

File: HW20/main.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        self.assertListEqual([f(0), f(1), f(2), f(3), f(4), f(5)],[0, 1, 1, 2, 3, 5])
+
+
+class TestFormattedName(unittest.TestCase):
```
Comment (mikolaz27) [inline HW20/main.py:37]: add more tests
